.travis.yml

```diff
@@ -1,3 +1,22 @@
 language: java
 jdk:
-  - oraclejdk8
\ No newline at end of file
+- oraclejdk8
+
+script:
+- ./gradlew check --stacktrace
```

Comment:
Use `build`. Otherwise you skip packaging and cause it to happen during the deploy step.